README.md

```diff
@@ -853,9 +853,9 @@ IEEE Trans. Image Process. (TIP), 31:7237-7251, 2022. [PDF](https://arxiv.org/ab
 - <a name="Zheng22tip"></a>Zheng, Y., Yu, Z., Wang, S., Huang, T.,  
 *[Spike-Based Motion Estimation for Object Tracking Through Bio-Inspired Unsupervised Learning](https://doi.org/10.1109/TIP.2022.3228168)*,  
 IEEE Trans. Image Process. (TIP), 32:335-349, 2022.
-- <a name="Gehrig22arxiv_flow"></a>Gehrig, M., Muglikar, M., Scaramuzza, D.,  
-*[Dense Continuous-Time Optical Flow from Events and Frames](https://arxiv.org/abs/2203.13674)*,  
-arXiv, 2022.  
+- <a name="Gehrig24pami"></a>Gehrig, M., Muglikar, M., Scaramuzza, D.,  
+*[Dense Continuous-Time Optical Flow from Events and Frames](https://doi.org/10.1109/TPAMI.2024.3361671)*,  
+IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2024., [PDF](https://arxiv.org/abs/2203.13674), [Code and Dataset] (https://github.com/uzh-rpg/bflow)
```

Comment:
there is an extra space between ] and (
![image](https://github.com/user-attachments/assets/4bbc7386-0ac1-41a3-bdc9-234a473cf4c2)